### PR TITLE
Issue #269 add a component to block multisig account from sending transactions.

### PR DESCRIPTION
**Pull request by NemTechCN**
add a component to block multisig account from sending transactions.
image

### CODE REVIEW

File: src/language/en-US.ts [hunk 1]
```diff
@@ -795,10 +795,10 @@ export const en_US = {
...
     proof: 'Proof',
     cosignatory: 'Cosignatory',
-    Add: 'Add', 
+    Add: 'Add',
     Remove: 'Remove',
     nickName:'Nick name',
     what_is_delegated_harvesting:'What is delegated harvesting ?',
     Delegated_harvesting_text_1:' Delegated harvesting enables an account to use a proxy private key that can be shared with a node securely. An eligible account may also delegate its importance score to a remote node for harvesting. In other words, you can use the importance score of your account to create new blocks without running a node. ',
-
+    Multisig_accounts_can_not_send_a_transaction_by_itself:'Multisig accounts can not send a transaction by itself.'
```
Comment (decentraliser) [inline src/language/en-US.ts]: by themselves

File: src/views/wallet/wallet-switch/WalletSwitchTs.ts [hunk 2]
```diff
@@ -146,9 +146,19 @@ export class WalletSwitchTs extends Vue {
...
         this.$refs.walletScroll["scrollTop"] = this.walletList
             .findIndex(({address}) => address === this.activeAddress) * 40
     }
+
+    mounted() {
+        this.scrollToActiveWallet()
+    }
+
+    @Watch('activeAddress')
+    onWalletChane(){
```
Comment (decentraliser) [inline src/views/wallet/wallet-switch/WalletSwitchTs.ts]: typo change

File: src/language/en-US.ts (hunk 1 above)
Comment (NemTechCN) [reply src/language/en-US.ts]: fixed

File: src/views/wallet/wallet-switch/WalletSwitchTs.ts (hunk 2 above)
Comment (NemTechCN) [reply src/views/wallet/wallet-switch/WalletSwitchTs.ts]: fixed

**Pull request merged**